backend/db_migrations.py

```
@@ -432,7 +539,16 @@ def initialize_schema(conn: sqlite3.Connection, force_recreate: bool = False) ->
         # Set schema version
         set_schema_version(conn, SCHEMA_VERSION)
         conn.commit()
-        
+
+        # Seed curated cancer taxonomy on a fresh DB so the v2 Cancer Research
+        # browse has data immediately. Idempotent — safe to re-run.
+        try:
+            from cancer_taxonomy_seed import seed_cancer_taxonomy
+            seeded = seed_cancer_taxonomy(conn)
+            print(f"🌱 Seeded {seeded['types']} cancer types, {seeded['subtypes']} subtypes")
+        except Exception as seed_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)